numpy/lib/twodim_base.py

```diff
@@ -136,12 +204,11 @@ def flipud(m):
     return m[::-1, ...]
 
 
-def rot90(m, k=1):
+def rot90(m, k=1, axes=(0,1)):
     """
-    Rotate an array by 90 degrees in the counter-clockwise direction.
+    Rotate an array by 90 degrees counter-clockwise in the plane specified by axes.
```

Comment:
changed the function description here [b51c013](https://github.com/numpy/numpy/commit/b51c0134c4b3df617d76b718871db7f9b2e6c6f1)